docs/spark/how-to-guides/connect-to-azure-storage.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Connect to Remote Storage from your local machine
+description: Connect to Azure Storage Accounts using .NET for Apache Spark from your local machine.
+ms.date: 09/24/2020
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+# Connect to Azure Data Lake Storage Gen 2 or WASB account
+
+In this article, you learn how to connect to an ADLS (Azure Data Lake Storage) Gen 2 or WASB (Windows Azure Storage Blob) account through an instance of [.NET for Apache Spark](https://github.com/dotnet/spark) running locally on your Windows machine.
+
+## Steps to setup the environment
```

Comment:
These steps are different from the generic setting up environment in the getting started guide since it requires installing Apache Spark without Hadoop and separate Hadoop installation.